### ISSUE TITLE
CT Designed API changes detected, please roll them back or revise the reference API

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/gravity-and-orbits#431]
```
gravity-and-orbits : phet-io-api-compatibility : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660539586646/gravity-and-orbits/gravity-and-orbits_en.html?continuousTest=%7B%22test%22%3A%5B%22gravity-and-orbits%22%2C%22phet-io-api-compatibility%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1660539586646%22%2C%22timestamp%22%3A1660544739761%7D&ea&brand=phet-io&phetioStandalone&phetioCompareAPI&randomSeed=332211
Query: ea&brand=phet-io&phetioStandalone&phetioCompareAPI&randomSeed=332211
Uncaught Error: Assertion failed: Designed API changes detected, please roll them back or revise the reference API:

PhET-iO Element missing: gravityAndOrbits.general.view.navigationBar.preferencesButton.preferencesDialogCapsule.archetype.preferencesPanels.generalPreferencesPanel.projectorModeCheckbox
Error: Assertion failed: Designed API changes detected, please roll them back or revise the reference API:
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660539586646/assert/js/assert.js:28:13)
at assert (phetioEngine.ts:365:22)
id: Bayes Puppeteer
Snapshot from 8/14/2022, 10:59:46 PM
```

**Comment by jessegreenberg** [comment]
This was caused by changes in https://github.com/phetsims/joist/issues/814, the checkbox was replaced by a toggle switch in the Preferences dialog. I re-generated the API but thought it would be good for a PhET-iO dev to confirm this is right.

IN https://github.com/phetsims/phet-io-sim-specific/commit/9b4403f1c3cd3005f92df44e1e3fb08cfb299720
- The checkbox was removed
- In the diff I saw references to the new `LOCALIZATION` EnumerationValue from https://github.com/phetsims/joist/issues/814

Finally, I think after this GlobalOptionsNode is no longer used in this sim and can be deleted.
